static/docs/commands-reference/import-url.md

```diff
@@ -119,195 +119,155 @@ downloaded file or directory from the external data source.
-Adding 'data/data.xml' to 'data/.gitignore'.
-Saving 'data/data.xml' to cache '.dvc/cache'.
-Saving information to 'data.xml.dvc'.
-```
-
-To track the changes with git run:
-
-```dvc
-$ git add data/.gitignore data.xml.dvc
+...
 ```
 
-> Note that it's possible to set up the other
-> [stages](/doc/commands-reference/run) from the _Getting Started_ example, but
-> since we don't need them for this example, we'll skip it.
+> Note that it's possible to set up the other stages from the _Get Started_
+> project, but since we don't need them for this example, we'll skip it.
 
 Let's take a look at the resulting stage file (DVC-file) `data.xml.dvc`:
 
 ```yaml
-deps:
-  - etag: '"f432e270cd634c51296ecd2bc2f5e752-5"'
-    path: https://dvc.org/s3/get-started/data.xml
 md5: 61e80c38c1ce04ed2e11e331258e6d0d
-outs:
-  - cache: true
-    md5: a304afb96060aad90176268345e10355
-    metric: false
-    path: data/data.xml
-    persist: false
 wdir: .
+locked: true
+deps:
+- etag: '"f432e270cd634c51296ecd2bc2f5e752-5"'
+  path: https://dvc.org/s3/get-started/data.xml
+outs:
+- md5: a304afb96060aad90176268345e10355
+  path: data/data.xml
+  cache: true
+  metric: false
+  persist: false
 ```
 
-The `etag` field in the DVC-file contains the ETag recorded from the HTTP
-request. If the remote file changes, the ETag changes, letting DVC know when the
-file has changed.
+The `etag` field in the DVC-file contains the
+[ETag](https://en.wikipedia.org/wiki/HTTP_ETag) recorded from the HTTP request.
+If the remote file changes, its ETag will be different, letting DVC know whether
+its necessary to download the it again.
 
 > See [DVC-File Format](/doc/user-guide/dvc-file-format) for more details on the
 > text format above.
 
+You may want to get out of and remove the `example-get-started` directory after
+trying this example (especially if trying out the following one).
+
 ## Example: Detecting remote file changes
 
 What if that remote file is one which will be updated regularly? The project
-goal might include regenerating a <abbr>data artifact</abbr> based on the
-updated data source. A pipeline can be triggered to re-execute based on a
-changed external dependency.
+goals might include regenerating a <abbr>data artifact</abbr> based on the
+updated data source. A [pipeline](/doc/commands-reference/pipeline) can be
+triggered to re-execute based on a changed external dependency.
 
-Let us again use the [Getting Started](/doc/get-started) example, in a way which
-will mimic an updated external data source.
+Let's use the [Get Started](/doc/get-started) project again, simulating an
+updated external data source.
 
-To make it easy to experiment with this, let us use a local directory as our
-remote data location. In real life the data file will probably be on a remote
-server, of course. Run these commands:
+To make it easy to experiment with this, let's use a local directory as our
+remote data location. (In real life, the data file will probably be on a remote
```

Comment:
`remote data` can we come up with a better term here? it can be confused with remote data storage

Reply:
How about:

"let's use a local machine directory (external to the sample DVC project) to simulate a remote data source location" ?

also updated in 1a214ea

Reply:
👍